test/dommock/DOMMockBuilder.ts

```diff
@@ -759,7 +759,11 @@ export default class DOMMockBuilder {
         if (mockBehavior.responseSuccess) {
```

Comment:
Could you simplify this `status()` getter by removing `mockBehavior.responseSuccess ` and returning your `mockBehavior.responseStatusCode`? It requires small changes in `MeetingSessionPOSTLogger.test.ts`.

```
get status(): number {
  return mockBehavior.responseStatusCode;
}
```

Reply:
Updated